website/docs/r/traffic_manager_endpoint.html.markdown

```diff
@@ -10,6 +10,8 @@ description: |-
 
 Manages a Traffic Manager Endpoint.
 
+~> **NOTE:** This resource is **deprecated** in favour of the `azurerm_traffic_manager_azure_endpoint`, `azurerm_traffic_manager_external_endpoint`, or `azurerm_traffic_manager_nested_endpoint` resources and will be removed in version 3.0 of the Azure Provider.
```

Comment:
The resource won't flag itself as deprecated until the beta flag is active, should the phrasing on this note reflect it is _going_ to be deprecated?

Reply:
Fixed by deprecating immediately 👍